platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -371,7 +371,7 @@ private boolean hasBundleLocation(Map<String, Object> service, Set<String> bundl
         .map(List.class::cast)
         .flatMap(List::stream)
         .map(Map.class::cast)
-        .map(m -> m.get("bundle_location"))
+        .map(m -> ((Map) m).get("bundle_location"))
```

Comment:
❓ Hmm this seems unnecessary. Did the upgrade really cost us the type information or was this an indexing glitch in IntelliJ?

Reply:
Even though there is a cast to `Map` right before, it still thinks `m` is an `Object`. I think this is because of how the stream inference is working type erasure with the original type.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project admin-core-appservice: Compilation failure
[ERROR] /C:/projects/ddf/platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java:[374,21] cannot find symbol
[ERROR]   symbol:   method get(java.lang.String)
[ERROR]   location: variable m of type java.lang.Object
```

Reply:
Interesting how the upgrade broke that. The error doesn't surprise me at all since you can lose type info really easily in streams. What surprises me is that it was fine prior to upgrading.